content/sigs/casc-sig/index.adoc

```diff
@@ -0,0 +1,75 @@
+---
+layout: sig
+title: "Configuration as Code"
+section: sigs
+sigId: "casc"
+logo: /images/user.gif
+tags:
+- configuration-as-code
+- jenkins-configuration-as-code
+- groovy
+- pipeline
+- yaml
+- gce
+- casc
+- casc-sig
+leads:
+- name: "Ewelina Wilkosz"
+  id: "ewelinawilkosz"
+  github: "ewelinawilkosz"
+participants:
+- name: "Oleg Nenashev"
+  id: "oleg_nenashev"
+  github: "oleg-nenashev"
+- name: "Tracy Miranda"
+  id: "tracymiranda"
+  github: "tracymiranda"
+- name: "Nicolas De loof"
+  id: "ndeloof"
+  github: "ndeloof"
+- name: "Liam Newman"
+  id: "Liam Newman"
+  github: "bitwiseman"
+
+organizations:
+- name: "CloudBees"
+  github: "cloudbees"
+
+links:
+  gitter: jenkinsci/casc-sig
+  googlegroup: jenkins-casc-sig
+meetings:
+  text: To be announced
+  link: TODO
+overview: >
+  The Configuration as Code (casc) group of contributors and collaborators focuses on improving Jenkins configuration as code related tools and practices.
+
+---
+
+The Configuration as Code (casc) group of contributors and collaborators focuses on improving Jenkins configuration as code related tools and practices.
+
+The group focuses on:
+
+* Jenkins Configuration as Code plugin
+* groovy hooks
+* Configuration management tools
+* Pipeline
```

Comment:
CC @svanoort @abayer @jennbriden Would you like to have Pipeline here? Or do you want to have a separate SIG?

Reply:
Yeah, I'm not sure it makes sense to lump Pipeline in with config-as-code as as a theme - it's a lot more than that.